docs/ci/WORKFLOW_OUTPUTS.md

```
@@ -223,3 +247,13 @@ jobs:
     steps:
       - run: echo "${{ needs.codex.outputs.final-message-summary }}"
 ```
+## Provider-neutral optional capability evidence
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)